test/networkd-test.py

```diff
@@ -96,10 +96,17 @@ def write_network_dropin(self, unit_name, dropin_name, contents):
         dropin_path = os.path.join(dropin_dir, "%s.conf" % dropin_name)
 
         os.makedirs(dropin_dir, exist_ok=True)
+        self.addCleanup(os.rmdir, dropin_dir)
         with open(dropin_path, 'w') as dropin:
             dropin.write(contents)
         self.addCleanup(os.remove, dropin_path)
 
+    def read_attr(self, link, attribute):
+        """Read a link attributed from the sysfs."""
+        self.assert_link_states(**{link:'managed'})
```

Comment:
actually no, I cannot use "nice" syntax. As I don't want to pass `{"link":"managed"}` kwarg, but i want to pass `{*link: "managed"}` kwarg, e.g. when `read_attr("eth0", "foo")` is passed, I want to call `assert_link_states(eth0="managed")`, not `assert_link_states(link="managed")`.